lib/ossert/stats/base.rb

```diff
@@ -89,6 +90,15 @@ def median(values, default_value: 0)
         (values[middle_idx - 1] + values[middle_idx]) / 2
       end
 
+      def get_count(metric)
```

Comment:
I know that my project suffers by lack of documentation, but let's try to add it to all new methods.
I also will try to fix that for previous code.